core/base/inc/TNamed.h

```diff
@@ -33,9 +33,9 @@ class TNamed : public TObject {
    TString   fTitle;           //object title
 
 public:
-   TNamed(): fName(), fTitle() { }
+   TNamed() = default;
```

Comment:
```suggestion
   TNamed() {}; // NOLINT: Can not use `= default` because of `TObject::kIsOnHeap` detection, see related discussion at https://github.com/root-project/root/pull/4320
```

Reply:
done.